### PR TITLE
remove imports

### PR DESCRIPTION
**Pull request by AllentDan**
Remove opset_imports that were not used after ONNX extraction.
Before:
image
After:
image

### CODE REVIEW

**Comment by grimoire** [comment]
What's this for?

**Comment by AllentDan** [comment]
> What's this for?

For backends like rknn-toolkit, `mmdeploy` domain influences the model conversion.

**Review by grimoire — APPROVED** [review]
Add detail comments about why do we need to remove imports.

File: docs/en/04-supported-codebases/mmocr.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ Please refer to [install.md](https://mmocr.readthedocs.io/en/latest/install.html
 
 Note that ncnn, pplnn, and OpenVINO only support the configs of DBNet18 for DBNet.
 
+CRNN model with TensorRT-int8 was only tested OK with TensorRT 7.2.3.4 and CUDA 10.2.
```
Comment (tpoisonooo) [inline docs/en/04-supported-codebases/mmocr.md:24]: 尽管我的也很水，但 “was only tested OK” 总感觉哪里不对。

**Review by tpoisonooo — APPROVED** [review]
LGTM.

**Pull request merged**